src/app/pages/send/send-crypto-asset-form/form/btc/btc-set-fee.tsx

```diff
@@ -0,0 +1,41 @@
+import { useLocation } from 'react-router-dom';
+
+import { Stack } from '@stacks/ui';
+import get from 'lodash.get';
+
+import { BitcoinSendFormValues } from '@shared/models/form.model';
+
+import { useRouteHeader } from '@app/common/hooks/use-route-header';
+import { ModalHeader } from '@app/components/modal-header';
+
+import { FeesCard } from '../../components/fees-card';
+import { useBtcSetFees } from './useBtcSetFees';
+
+function useBtcSetFeeState() {
+  const location = useLocation();
+  return {
+    txValues: get(location.state, 'values') as BitcoinSendFormValues,
+  };
+}
+
+export function BtcSetFee() {
```

Comment:
I think the naming throughout this PR needs to accommodate ordinal txs as well, so using `Bitcoin` in naming rather than `Btc` ...maybe to match nav `toChooseBitcoinFee` could be `BitcoinChooseFee` (and route to match)? Also, I think this needs to be located in the `Family` folder and reused for ordinal sends (see attached issue)? It might be that we need to move that form first to make sharing components b/w these forms work?

Reply:
yeah, agree here